src/witan/models/dem/ccm/fert/fertility_mvp.clj

```diff
@@ -115,8 +115,8 @@
    (cf/jumpoffyr-method-average historic-asfr
                                 :fert-rate
                                 :fert-rate
-                                1
-                                (inc fert-last-yr))})
+                                2013
+                                2014)})
```

Comment:
It's equivalent to what it was doing before. And I thought it unlikely to change (for now) bc the ONS projected births by age of mother data is for for 2013.
But you're right I'll change that!